### ISSUE TITLE
Remove deprecated CustomPeriodicEvent (part 2)

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/ignite#1375]
## 🚀 Feature

A follow-up issue to https://github.com/pytorch/ignite/issues/1247 and the PR https://github.com/pytorch/ignite/pull/1252 where we forgot about this notebook example : https://github.com/pytorch/ignite/blob/master/examples/notebooks/EfficientNet_Cifar100_finetuning.ipynb - Cell 40.

Fixing this issue can be an opportunity to learn about Ignite's API and neural networks.

For Hacktoberfest contributors, feel free to ask questions for details if any and say that you would like to tackle the issue.
Please, take a look at CONTRIBUTING guide.

PS. there is already an [open PR on that](https://github.com/pytorch/ignite/pull/1255) but it does not fix the issue and the contributor lacks of time to continue. It will be closed in favor of a new correct PR.

**Comment by trsvchn** [comment]
@vfdev-5 I would like to investigate ignite api, can try this issue?

**Comment by vfdev-5** [comment]
Sure @trsvchn , thanks !

**Comment by vfdev-5** [comment]
Hi @trsvchn how is going your investigations of Ignite's API and the notebook ? 
Feel free to ask if you need a help/hints/explanations etc.

**Comment by aneeshnema** [comment]
@vfdev-5 if no one is working on this I would like to contribute.

**Comment by trsvchn** [comment]
Hi @vfdev-5 sorry for the delay.
> how is going your investigations of Ignite's API and the notebook

Yes, it was hard a little, I am not an OOP person, but I think I'm beginning to see :))

If I understood correctly, after removing `CustomPeriodEvent` related lines we need just smth like this:

```python
...
trainer.add_event_handler(Events.EPOCH_STARTED(every=3), run_evaluation)
...
```

And, it looks like `OutputHandler` has newer api: `global_step_transform` arg instead of `another_engine`.

I am not sure, should we use here `global_step_transform=global_step_from_engine(trainer)`?
Does this looks fine?

```python
# Log train eval metrics:
tb_logger.attach_output_handler(
    train_evaluator,
    event_name=Events.EPOCH_COMPLETED
    tag="training",
    metric_names=list(metrics.keys()),
    global_step_transform=global_step_from_engine(trainer)
)

# Log val metrics:
tb_logger.attach_output_handler(
    evaluator,
    event_name=Events.EPOCH_COMPLETED,
    tag="test",
    metric_names=list(metrics.keys()),
    global_step_transform=global_step_from_engine(trainer))
)
```

**Comment by vfdev-5** [comment]
@trsvchn yes, `CustomPeriodEvent` is simply replaced by filtered event like `Events.EPOCH_STARTED(every=3)`.

> And, it looks like OutputHandler has newer api: global_step_transform arg instead of another_engine.

Thanks for catching that too. I thought we fixed that everywhere. Could you please also make sure that we now using `` everywhere instead of `another_engine` ?

Yes, `global_step_transform=global_step_from_engine(trainer)` is a correct replacement.


Btw, you can run the notebook on the Colab: https://colab.research.google.com/github/pytorch/ignite/blob/master/examples/notebooks/EfficientNet_Cifar100_finetuning.ipynb to manually check your updates.

**Comment by vfdev-5** [comment]
@aneeshnema thanks for proposing help, but this issues is assigned to @trsvchn who is working on it.
Please, take a look at the list of other Hacktoberfest issues : https://github.com/pytorch/ignite/issues?q=is%3Aissue+is%3Aopen+label%3AHacktoberfest+no%3Aassignee
and take the one you'd like to work on.

**Comment by trsvchn** [comment]
@vfdev-5 Thanks for reviewing the proposed changes! Yes, I was using colab.

> Could you please also make sure that we now using `` everywhere instead of another_engine ?

Sure! I will check that!

**Comment by trsvchn** [comment]
@vfdev-5 No, I didn't find any `another_engine`.
